addons/mail/static/src/core/common/store_service.js

```diff
@@ -621,6 +624,21 @@ export class Store extends BaseStore {
             messages: this["mail.message"].insert(messages),
         };
     }
+    reset() {
+        super.reset();
+        this.channels.status = "not_fetched";
```

Comment:
Should be applied to all `makeCachedFetchData`, so also canned reponses and livechatChannels